src/Components/Publishing/RelatedArticles/ArticleCards/ArticleCard.tsx

```diff
@@ -75,18 +75,19 @@ export class ArticleCard extends Component<Props> {
     const { article } = this.props
     const mediaDate = getMediaDate(article)
     const date = article.layout === "video" ? mediaDate : article.published_at
+    const formatedDate = new Date(date).toISOString()
```

Comment:
👍 

I interpret this to mean that this article `date` attribute used to come back from Positron as an ISO date — but then started coming back in a different format because of a moment/luxon switch somewhere upstream. Does that sound about right?

Reply:
Yep!

Reply:
See #2554 for another instance of this!